airbyte-integrations/connectors/source-amplitude/acceptance-test-config.yml

```diff
@@ -1,24 +1,35 @@
 connector_image: airbyte/source-amplitude:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
+    tests:
     - spec_path: "source_amplitude/spec.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.17"
   connection:
+    tests:
     - config_path: "secrets/config.json"
       status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
       status: "failed"
   discovery:
+    tests:
     - config_path: "secrets/config.json"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/streams_with_output_records_catalog.json"
-  incremental:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog_without_events.json"
-      # Unable to use 'state_path' because Amplitude returns an error when specifying a date in the future.
-      # future_state_path: "integration_tests/abnormal_state.json"
+    tests:
+      - config_path: "secrets/config.json"
+        expect_records:
+          path: "integration_tests/expected_records.jsonl"
+        empty_streams:
+          - name: "cohorts"
+            bypass_reason: "This stream is empty due to free subscription plan for the sandbox."
+          - name: "annotations"
+            bypass_reason: "This stream is empty due to free subscription plan for the sandbox."
```

Comment:
not blocking, but what would it cost to have a plan that allows us to use these streams?

Reply:
@igrankova do we have the answer to this question?

Reply:
@evantahler, these 2 streams are on the paid plan and we are working on obtaining access to them. I'll let you know when we receive it.
cc @bazarnov, @YowanR